db.go

```diff
@@ -550,7 +550,7 @@ func (db *DB) reload(deleteable ...string) (err error) {
 	if len(blocks) == 0 {
 		return nil
 	}
-	maxt := blocks[len(blocks)-1].Meta().MaxTime
+	maxt := blocks[len(blocks)-1].Meta().MaxTime + 1
```

Comment:
is that really needed?

Reply:
Ok, it is, because if block A is `0-10` and block B is `10-20`, sample t=10 must be in both.

However, this is quite risky change, affecting lots of things. Wonder if we can abort this and try to make compactor deduplicating these chunks correctly. Looking into that.

Reply:
Ok, talked offline with @fabxc and in fact we can kill this change and leave just adjusting the nextAt change.

This is because:
> if a block has `minTime: 10` and `maxTime: 20` the intend was for this to be interval `[10,20)`